tests/Agency/AddFollowerTest.cpp

```diff
@@ -49,15 +49,15 @@ namespace arangodb {
 namespace tests {
 namespace add_follower_test {
 
-const std::string PREFIX = "arango";
-const std::string DATABASE = "database";
-const std::string COLLECTION = "collection";
-const std::string SHARD = "s99";
-const std::string SHARD_LEADER = "leader";
-const std::string SHARD_FOLLOWER1 = "follower1";
-const std::string SHARD_FOLLOWER2 = "follower2";
-const std::string FREE_SERVER = "free";
-const std::string FREE_SERVER2 = "free2";
+[[maybe_unused]] const std::string PREFIX = "arango";
```

Comment:
Some variable unused in agency test.
It's produce warning with my compiler, which is werror.
So I marked all their constants as maybe_unused

Reply:
Maybe just remove unused constants?

Reply:
I don't want to touch this